.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+name: Node.js CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    env:
+      working-dir: ./src
```

Comment:
Isn't `working-dir` the directory where the `package.json` should be?

Reply:
Hmm... good question, I don't know. I'll double-check tho, thanks!

Reply:
👍 yes, that appears to be the case. I'll fix this now.